logstash-core/lib/logstash/config/source/local.rb

```diff
@@ -102,7 +102,7 @@ def get_unmatched_files
       end
 
       def valid_encoding?(content)
-        content.ascii_only? && content.valid_encoding?
+        content.ascii_only? || content.valid_encoding?
```

Comment:
@jsvd 

> is there a situation where the encoding is valid but it's not ascii, or it is ascii but not valid? maybe content.valid_encoding? is enough.

I know too little about Ruby to answer that I'm afraid :( Instinctively it seems like `content.valid_encoding?` should imply `valid_encoding? == true`.
=> Valid and not ascii is what was reported in the issue and fixed here, the other way around ascii and not valid should be impossible?